src/App.css

```diff
@@ -16,3 +16,6 @@
   border-bottom-right-radius: 0px;
 }
 
+a {
```

Comment:
Why do you need this rule?

Reply:
The issue and commit titles have the links to them. Without this rule, they would be underline.

Reply:
Then the `<InforCard>` component should make sure it's title is not underlined. You shouldn't apply a generic rule for all `a` in the page.